static/js/stream_data.js

```diff
@@ -515,6 +515,12 @@ export function can_change_permissions(sub) {
     return page_params.is_admin && (!sub.invite_only || sub.subscribed);
 }
 
+export function can_view_subscribers(sub) {
+    // User can add other users to stream if stream is public or user is subscribed to stream.
+    // Guest users can't access subscribers of any(public or private) non-subscribed streams.
```

Comment:
I know you only moved this comment, but it appears to be wrong for this block -- we should explain under what circumstances the user can view subscribers.

Reply:
(Or maybe just move it outside so it more clearly applies to both this function and the next one (which is about who can change subscribers)